robot/EDA/resources/AffiliationsPageObject.py

```diff
@@ -76,13 +75,7 @@ def open_item(self, locator, error_message, capture_screen):
             timeout=60,
             error=error_message
         )
-#        self.selenium.click_element(locator)
-
-        self.selenium.driver.execute_script(
-            "arguments[0].click()", 
-            self.selenium.driver.find_element_by_xpath(locator)
-        )
-
+        self.selenium.click_element(locator)
 
         if capture_screen:
```

Comment:
Are you trying to capture a screenshot when there is an error? If yes u can use the property @capture_screenshot_on_error on the keyword and it will take care of the screenshot. If you are capturing a screenshot for specific scenario then it is better to provide those details in the keyword documentation

Reply:
It's an arg value - so I'm only capturing a screenshot as passed to the function, which in production is always passed in as 'False'.  During debugging, all I have to do is toggle the arg to be 'True' and then I can a nifty screenshot :-)

Reply:
Then maybe provide some info in comments to say pass capture_screen as False to turn off screen capture and  True to turn on screen capture. Also, if u mention 
"@capture_screenshot_on_error    
    def open_item(self, locator, error_message):" it will capture screenshot only if test case fails on this keyword (which is useful for debugging)and you wouldn't have to worry about it as additional step to ur keyword. So please see if that meets ur purpose

Reply:
Sounds good - I'll explain the purpose of the parameter.